examples/research_projects/diffusion_dpo/train_diffusion_dpo_sdxl.py

```diff
@@ -898,6 +936,10 @@ def collate_fn(examples):
                 timesteps = torch.randint(
                     0, noise_scheduler.config.num_train_timesteps, (bsz,), device=latents.device, dtype=torch.long
                 ).repeat(2)
+                if args.is_turbo:
+                    # Learn a 4 timestep schedule
+                    timesteps_0_to_3 = timesteps % 4
+                    timesteps = 250 * timesteps_0_to_3 + 249
```

Comment:
Can this be explained in the comments a bit? I know about the 4-step part that was discussed in the ADD paper for training of the student.